### DISCUSSION TITLE
Alas has been incorrectly configured

### DISCUSSION THREAD

**Discussion by ABCDEFG2460** [LmeSzinc/AzurLaneAutoScript#2709] [Q&A]
My Alas has been incorrectly configured. I have tried to correct the deploy file, and I have also changed three Chinese mirror sources, which have been in recent days. Please help me

**Comment by LmeSzinc** [comment]
You don't need those CN mirrors if you are not in China mainland.

**Reply by ABCDEFG2460** [reply]
I am in CN, so I use Chinese mirror image, but I have been unable to update it in recent days, sad

**Comment by ChenLily1019** [comment]
Please change the Repository in the config/deploy.yaml file to this
Repository: 'git://git.lyoko.io/AzurLaneAutoScript'

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2687#issuecomment-1572526474
